c7n/resources/glue.py

```diff
@@ -80,3 +81,57 @@ def delete_connection(self, r):
     def process(self, resources):
         with self.executor_factory(max_workers=2) as w:
             list(w.map(self.delete_connection, resources))
+
+
+@resources.register('glue-dev-endpoint')
+class GlueDevEndpoint(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'glue'
+        enum_spec = ('get_dev_endpoints', 'DevEndpoints', None)
+        detail_spec = None
+        id = name = 'EndpointName'
+        date = 'CreatedTimestamp'
+        dimension = None
+        filter_name = None
+
+    permissions = ('glue:GetDevEndpoints',)
+
+
+@GlueDevEndpoint.action_registry.register('delete')
+class DeleteDevEndpoint(BaseAction):
+    """Deletes public Glue Dev Endpoints
+
+    :example:
+
+    .. code-block: yaml
+
+        policies:
+          - name: delete-public-dev-endpoints
+            resource: glue-dev-endpoint
+            filters:
+              - PublicAddress: present
+            actions:
+              - type: delete
+    """
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteDevEndpoint',)
+
+    def delete_dev_endpoint(self, client, endpoint_set):
+        try:
+            for e in endpoint_set:
+                client.delete_dev_endpoint(EndpointName=e['EndpointName'])
```

Comment:
the try except should be around the inner loop, else a resumable error on one would stop the rest of the set.

Reply:
Ah crap, I will get that corrected